torchvision/io/image.py

```diff
@@ -148,24 +148,34 @@ def write_png(input: torch.Tensor, filename: str, compression_level: int = 6):
     write_file(filename, output)
 
 
-def decode_jpeg(input: torch.Tensor, mode: ImageReadMode = ImageReadMode.UNCHANGED) -> torch.Tensor:
+def decode_jpeg(input: torch.Tensor, mode: ImageReadMode = ImageReadMode.UNCHANGED,
+                device: str = 'cpu') -> torch.Tensor:
```

Comment:
Just flagging that there are other methods such as the `read_image()` and `decode_image()` to which you can also expose the GPU support. If you choose not to introduce the `device` argument to them, they will only decompress JPEG on CPU. Personally, I agree with your approach and I would also keep the feature contained for now. Once it becomes stable it would be very easy to expose it via the other APIs in separate PRs.